### PR TITLE
Stuff

### LINKED ISSUE #13: Chat prompt-response loop
Creation of the Chat prompt-response loop.

### LINKED ISSUE #14: main.py commandline arguments and input handling
The main.py should allow the user to pass a file path as a commandline argument.
Will use argparse for this, with -i / --input argument.
Optionally, the user may use input redirection like python3 main.py < ./dir/file.txt

### PR DESCRIPTION
**Pull request by andreicozma1**
Added main argument parser, implemented chat loop, added utils for processing and response variation, added regexes, alternatives, and stopwords to store

### CODE REVIEW

File: lib/chat.py [hunk 1]
```diff
@@ -6,5 +6,168 @@
...
+        ["questions", "prompts", "queries"],
+        ["you can ask", None],
+        ":\n",
+        [f'  - "{ex}"' for ex in random.sample(store.example_prompts, 3)],
+    ),
+    # TODO: Add regex and responses here
+}
+
+
+def preprocess_msg(msg: str) -> str:
```
Comment (zac-perry) [inline lib/chat.py:113]: Hey! This pre and post processing is for the user input message to the chat bot right? I was working on the processing for the text file's and I think our functions will overlap some (which is nbd at all) but I just wanted to double check

Comment (Manan-dev) [reply lib/chat.py:113]: I think Andrei has that functionality implemented in `utils.py`. But if you have code that improves it/adds additional functionality, please go ahead and comment it here or create a PR after this is merged!

Comment (zac-perry) [reply lib/chat.py:113]: Okay, will do! Thanks!

**Review by zac-perry — APPROVED** [review]
LGTM

File: lib/chat.py (hunk 1 above)
Comment (andreicozma1) [reply lib/chat.py:113]: yeah this is for the user input, though the functions it uses are in utils like Manan said. We can keep adding to the utils ones and make improvements there

**Pull request merged**